themes/orbit/layout/index.ejs

```diff
@@ -11,96 +12,125 @@
     </div>
     <div class="centered">
       <h1>Orchestrate your data.</h1>
-      <div class="install-command">
-        <span class="code">npm install @orbit/data</span>
-      </div>
       <p>
         <a class="button white" href="/v<%= config.versions.current %>/guide/">GET STARTED</a>
       </p>
     </div>
   </div>
 </div>
 
-<div id="tour">
+<div class="tour">
   <div class="inner">
-    <section>
-      <h3>Orbit is a framework for orchestrating access, transformation,
-        and synchronization between data sources.</h3>
+    <section class="tour__masthead">
+      <div class="tour__masthead__intro">
+        <h2>A flexible client and server side ORM</h2>
+        <p>Orbit is a composable data framework, helping you to orchestrate access,
+          transformation, and synchronization between data sources.</p>
+      </div>
+      <div class="tour__masthead__code-example">
+        <%- partial('partials/code_sample') %>
+        <div class="text-right">
+          <a href="/v<%= config.versions.current %>/guide/" class="learn-more" style="padding-right: 5px;">
+            learn more <i class="fas fa-arrow-right"></i>
```

Comment:
All of the "learn more" links just link to the guide homepage.  Should these link to individual/related sections of the guide?